craft_application/fetch.py

```diff
@@ -294,6 +278,28 @@ def configure_instance(
     return net_info.env
 
 
+def get_log_filepath() -> pathlib.Path:
+    """Get the path containing the fetch-service's output."""
+    # All craft tools log to the same place, because it's a single fetch-service
+    # instance. It needs to be a location that the fetch-service, as a strict
+    # snap, can write to.
+    logdir = _get_service_base_dir() / "craft-logs"
+    logdir.mkdir(exist_ok=True, parents=True)
+    return logdir / "fetch-service.log"
```

Comment:
Since this is a fixed path and the feature is experimental maybe we could not announce it to the user just yet. If someone reports a bug we can ask for the log dump.